app/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion "29.0.2"
     defaultConfig {
         applicationId "com.eis.networkproject"
-        minSdkVersion 15
+        minSdkVersion 26
```

Comment:
What forced this big change in the minimum sdk version? I believe around 20% of devices would not be able to use the app requiring this library, more people than you might expect still have Android 6 or 7. If it's for a very solid reason then I think it could be ok.

Reply:
I don't specifically remember why the sdk was increased from 15 to 26 (it happened 2-3 weeks ago afterall), but I think it's because of the smslibrary. Anyway this sdk update is in the app gradle, not the library gradle, so it shouldn't affect the library at all

Reply:
Right I didn't notice it was in the demo module, my bad